actor/nats-pub/Makefile

```diff
@@ -0,0 +1,21 @@
+# Makefile for nats-pub
+
+PROJECT  = nats_pub
+VERSION  = $(shell cargo metadata --no-deps --format-version 1 | jq -r '.packages[] .version' | head -1)
+REVISION = 0
+# list of all contract claims for actor signing (space-separated)
+CLAIMS   = wasmcloud:httpserver wasmcloud:messaging
```

Comment:
Should we add `wasmcloud:builtin:logging`, or is unnecessary/confusing?

Reply:
I don't think we need the builtin logging for anything